tailwind.config.js

```diff
@@ -10,6 +10,7 @@ export default {
       fontFamily: {
         primary: ['BakerSignetStd', 'sans-serif'],
         secondary: ['Unique', 'sans-serif'],
+        inherit: ['inherit'],
```

Comment:
🧐 why?

Reply:
![image](https://github.com/voyageglobal/voyage-frontend/assets/170460349/b0755080-bbd8-4c0c-8079-aff02c45eba6)

Because i don't see inherit family already in TW, Now I think, that it'll be maybe better to use font-fourth (its shorter by 1 symbol, lol)

Reply:
anyway, I don't really like the approach when you set the font for the whole section and then inherit it in text blocks

it would be more reliable to set font only for text elements to avoid implicit inheritance

Reply:
In addition, I don't see any visual changes by removing `font-inherit` from elements because it seems this css prop has inheritance by default